Sources/SQLKit/Builders/SQLDropTableBuilder.swift

```diff
@@ -27,6 +27,13 @@ public final class SQLDropTableBuilder: SQLQueryBuilder {
         dropTable.ifExists = true
         return self
     }
+
+    /// The optional `IF EXISTS` clause suppresses the error that would normally
+    /// result if the table does not exist.
+    public func behaviour(_ behaviour: SQLDropBehaviour) -> Self {
```

Comment:
Two helper methods `cascade()` and `restrict()` that call into this would improve readability.